Task3/js/app.js

```diff
@@ -0,0 +1,42 @@
+const container = document.querySelector('.zombies');
+const status = document.querySelector('.zombies-status');
+const numberZombie = document.querySelectorAll('.zombies-status span')[0];
+const allZombies = document.querySelectorAll('.zombies-status span')[1];
+let counter = 0;
+let zombie;
+
+allZombies.textContent = zombies.length;
+numberZombie.textContent = counter + 1;
+
+function createZombie() {
+    if (zombies.length && zombies.length - 1 >= counter) {
+        const zombieElement = new ZombieElement();
+
+        if (zombies[counter].type === ZOMBIE_TYPE.SMALL) {
```

Comment:
this will not work if you add one "ZombieSmall" at the end of the "zombies" array

Reply:
Why? It is working, even if I add several "ZombieSmall" or other types of zombies